__tests__/article.test.js

```diff
@@ -88,3 +87,69 @@ describe("04 GET /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("05 GET /api/articles", () => {
+  describe("Functionality", () => {
+    test("Should return an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.constructor).toEqual(Array);
+          expect(articles.length).toEqual(13);
+          articles.forEach((article) => {
+            expect(article.constructor).toEqual(Object);
+          });
+        });
+    });
```

Comment:
Your assertions in the following 2 tests are checking the length and the forEach and are covering the fact that it's an array, so the assertion on line 99 is redundant here.